### ISSUE TITLE
linker errors if multiple C files have same name

### ISSUE THREAD

**Issue by Waqar144** [c3lang/c3c#1592]
when using c sources mixed with c3 sources, if there are two C files in different directories with same name then it results in linker errors

**Comment by lerno** [comment]
Yes, that's probably rather inevitable using the current scheme where all .o files are placed in the same directory. Possibly we could rename them?

**Comment by Waqar144** [comment]
Yes, that'd be good. I'd suggest just suffixing a number for the duplicates. The compiler can print note to stderr about the duplication and the new name.
